gcc/testsuite/rust/compile/invalid_utf8

```diff
@@ -0,0 +1 @@
+�
```

Comment:
I'm really not well versed on UTF-8. Can you explain why this is invalid?

Reply:
It is an 0xff byte followed by an 0x0a byte. 0xff is not a valid part of any UTF-8 encoded character. The highest allowed byte is 0xf7 which would theb have to be followed by 4 continuation bytes. Note that your browser may render this sequence as ÿ. For example for me it does when showing the files, but not when showing the discussion. This is caused by the browser autodetecting a different character encoding than UTF-8.

Reply:
Ah, thanks @bjorn3 :) yeah I was confused because it does render as `ÿ` in the Files, but I hadn't checked back here. Thanks!